src/In.ProjectEKA.HipLibrary/Patient/Model/DiscoveryRepresentation.cs

```diff
@@ -9,4 +12,25 @@ public DiscoveryRepresentation(PatientEnquiryRepresentation patient)
             Patient = patient;
         }
     }
+
+    public class GatewayDiscoveryRepresentation
+    {
+        public PatientEnquiryRepresentation Patient { get; }
+        public Guid RequestId { get; }
+        public string Timestamp { get; }
+        public string TransactionId { get; }
+        public Error Error { get; }
+        public Resp Resp { get; }
+
+        public GatewayDiscoveryRepresentation(PatientEnquiryRepresentation patient, Guid requestId,
+            string timestamp, string transactionId, Error error, Resp resp)

```

Comment:
Does not need to be at HIP library as response of gateway representation/ discovery representation is prepared by HIP service.